### PR TITLE
Fix ingeset pipeline path for system datasets

### PR DESCRIPTION
**Pull request by ruflin**
The auth and syslog system dataset contained a wrong path to the ingest pipeline which meant it is not loaded. This fixes it.

### CODE REVIEW

File: util/dataset.go [hunk 1]
```diff
@@ -16,6 +16,10 @@ import (
 	"github.com/elastic/go-ucfg/yaml"
 )
 
+const (
+	DirIngestPipeline = "ingest-pipeline"
```
Comment (ruflin) [inline util/dataset.go:20]: @mtojek I'm tempted to say we should rename this to `ingest_pipeline` but don't have a strong argument for either. Any opinion?

**Comment by ruflin** [comment]
@neptunian This fixes the logs for the system package.

File: dev/import-beats/packages.go [hunk 2]
```diff
@@ -324,7 +324,7 @@ func (r *packageRepository) save(outputDir string) error {
 
 			// dataset/elasticsearch
 			if len(dataset.elasticsearch.ingestPipelines) > 0 {
-				ingestPipelinesPath := filepath.Join(datasetPath, "elasticsearch", "ingest-pipeline")
+				ingestPipelinesPath := filepath.Join(datasetPath, "elasticsearch", util.DIR_INGEST_PIPELINE)
```
Comment (mtojek) [inline dev/import-beats/packages.go:327]: is this correct path (caps)?

File: util/dataset.go (hunk 1 above)
Comment (mtojek) [reply util/dataset.go:20]: I'm fine with this path.

File: dev/import-beats/packages.go (hunk 2 above)
Comment (ruflin) [reply dev/import-beats/packages.go:327]: :-( No. Seems like my IDE did not catch this during refactoring :-(

File: util/dataset.go (hunk 1 above)
Comment (ruflin) [reply util/dataset.go:20]: Which one, `-` or `_`?

Comment (mtojek) [reply util/dataset.go:20]: I mean, I'm with both, so also with the underscore :)

Comment (ruflin) [reply util/dataset.go:20]: I was hoping you would have a preference to tip it in one direction :-)

**Pull request merged**